module/Accounting/src/Accounting/Entity/Transaction.php

```diff
@@ -5,19 +5,25 @@
 use Application\Entity\DomainEntity;
 
 /**
- * @ORM\Entity @ORM\Table(name="account_transactions")
+ * @ORM\Entity @ORM\Table(name="transactions")
```

Comment:
Il nome della tabella lascialo `account_transactions` perché `transactions` è troppo generico, ho paura di collisioni in futuro